test-suite/src/test/scala/org/scalajs/testsuite/javalib/CollectionTest.scala

```diff
@@ -165,7 +165,7 @@ trait CollectionTest extends JasmineTest with ExpectExceptions {
       coll.add("one")
       expect(coll.contains("one")).toBeTruthy
       expect(coll.contains("two")).toBeFalsy
-      expect(coll.contains(null)).toBeFalsy
+      //expect(coll.contains(null)).toBeFalsy
```

Comment:
ok for ```def allowsNullElementQuery: Boolean```
but for ```def allowsNullElement: Boolean``` clash with the one in SetFactory, hopefully it will be another pr topic